frontend/src/static/js/static_pages/styles/AddMediaPage.scss

```
@@ -616,3 +617,71 @@ $media-upload-item-name-line-height: 20px;
+}
+
+.add-media-fine-uploader {
+	width: 100%;
+}
+
+.add-media-fine-uploader-hidden-button {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	overflow: hidden;
+	clip: rect(0 0 0 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)